charts/nutanix-csi-storage/values.yaml

```diff
@@ -95,12 +97,12 @@ servicemonitor:
 #
 
 provisioner:
-  image: quay.io/karbon/ntnx-csi:v2.4.1
+  image: quay.io/karbon/ntnx-csi:v25_test
```

Comment:
Do you plan to check this in?

Reply:
Lets wait until I publish ntnx-csi:v2.5.0.

Reply:
Correct I'm waiting final version